client/data/disputes/hooks.ts

```diff
@@ -47,7 +47,7 @@ export const useDispute = (
 	);
 
 	const { acceptDispute } = useDispatch( STORE_NAME );
-	const doAccept = () => acceptDispute( id );
+	const doAccept = () => acceptDispute( dispute );
```

Comment:
This `doAccept` function returned by the `useDispute` hook can be removed once `LegacyDisputeDetails` is removed, since it is the only consumer of this function.

https://github.com/Automattic/woocommerce-payments/blob/3d4d7a594cf4681a71b4c6b41971b0b67bf50749/client/disputes/details/index.tsx#L31

Reply:
I've resolved this in https://github.com/Automattic/woocommerce-payments/pull/7458#issuecomment-1758791566

Reply:
Nice. Thank you, @Jinksi !